pixel-point/pixelpoint-website
# fix: add fix to Configu cs: remove openSource

**Describe what changes this pull request brings**
This pull request adds fix to Configu case study: removes openSource feature & GitHubStars.

**Steps to test**

1. Open the [page](https://pixelpointwebsite-configucsfix.gatsbyjs.io/case-studies/configu/)
2. Confirm that everything looks and works as expected.
